XSLT/nashvillep15769coll19qdctomods.xsl

```diff
@@ -0,0 +1,332 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:oai_qdc="http://worldcat.org/xmlschemas/qdc-1.0/"
+                xmlns:dcterms="http://purl.org/dc/terms/"
+                xmlns:dc="http://purl.org/dc/elements/1.1/"
+                xmlns:oai="http://www.openarchives.org/OAI/2.0/"
+                xmlns="http://www.loc.gov/mods/v3"
+                xpath-default-namespace="http://worldcat.org/xmlschema/qdc-1.0/"
+                exclude-result-prefixes="xs xsi oai oai_qdc dcterms dc"
+                version="2.0">
+
+  <!-- output settings -->
+  <xsl:output encoding="UTF-8" method="xml" omit-xml-declaration="yes" indent="yes"/>
+  <xsl:strip-space elements="*"/>
+
+  <!-- includes and imports -->
+
+  <!--
+    Collection/Set = UT Chattanooga's sets as Qualified Dublin Core => MODS
+  -->
+
+  <!-- variables and parameters -->
+  <!--
+    dc:language processing parameter: there are multiple language values in the
+    QDC.
+  -->
+  <xsl:variable name="catalog" select="document('catalogs/utc_catalog.xml')"/>
```

Comment:
change to `'catalogs/nashville_catalog.xml'`

Reply:
Oh god, yeah